### PR TITLE
Adding test for capacity() logic

### PR DESCRIPTION
**Pull request by blvns**

### CODE REVIEW

File: density/density.py [hunk 1]
```diff
@@ -329,6 +329,13 @@ def capacity():
         cap_data = json.load(json_data)['data']
     # Read current data
     cur_data = db.get_latest_data(g.cursor)
+    locations = calculate_capacity(cap_data, cur_data)
+    return render_template('capacity.html', locations=locations)
+
+#Calculates capacity with cap_data and cur_data and puts with respective
```
Comment (natebrennand) [inline density/density.py]: These comments should be moved to a [docstring](https://www.python.org/dev/peps/pep-0257/) in the lines after the `def`.
See the other functions defined in this file for an example

**Comment by natebrennand** [comment]
This is awesome!

I don't think you should test against the values that are loaded from the JSON file because then your test values are tied to those values. If you override the `cap_data` dictionary in your test then you can do this safely.

**Comment by jzf2101** [comment]
Yay :+1:

File: density/tests/test_calculate_capacity.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+from density import calculate_capacity
+
+
+class TestCalculateCapacity(unittest.TestCase):
+
+    # testing logic in calulate_capacity()
+    def test_calculate_capacity(self):
+        # test version of cap_data
+        cap_test = [{'capacity': 100, 'group_name': 'Butler Library 6'},
```
Comment (blvns) [inline density/tests/test_calculate_capacity.py]: @natebrennand I'm getting pep8 compliance issues about the lines here - if I don't like up the entries (indent more), I get an indentation error, but when I indent like they ask for pep8 complains about the lines being too long. Is this a problem/ should I worry about it?
screen shot 2015-04-03 at 4 29 09 pm

**Comment by natebrennand** [comment]
You need to merge in master again so that they tests can run.

**Pull request merged**